README.md

```diff
@@ -181,6 +184,82 @@ Events caused by pointer movement or scrolling do not count as "input" for the
 purpose of the recent input exclusion and the input-related attributes on the
 `LayoutShift` entry.
 
+### Source Attribution
+
+_NOTE: The `sources` attribute is not yet implemented in any browser, and its
+definition is still under active development._
+
+On a complex website, it can be difficult to understand the cause of a high
+CLS score given only the numeric values in the `value` attribute of the
+`LayoutShift` entries.
+
+The `sources` attribute connects the `LayoutShift` back to the specific DOM
+elements that experienced the shift, so that the developer has more insight
+into the causes of layout instability on their site.
+
+The `sources` attribute is an array of up to 5 `LayoutShiftAttribution` objects:
+
+```idl
+interface LayoutShiftAttribution {
+    Node node;
```

Comment:
This may need to change depending on how the spec ends up.

Reply:
Ack - but I'd prefer to land this PR without blocking on the spec changes, if only to unblock the TAG review.